packages/tools/fetch-tool/src/fluidFetchSharePoint.ts

```diff
@@ -61,7 +61,8 @@ export async function resolveWrapper<T>(
             // Re-auth
             return resolveWrapper<T>(callback, server, clientConfig, true, forToken);
         }
-        return Promise.reject(new Error(`Fail to connect to ODSP server\nError Response:\n${e}`));
+        e.message = `Fail to connect to ODSP server\nError Response:\n${e.message}`;
+        return Promise.reject(e);
```

Comment:
Couple nits - I don't like the existing "Error Response" term, maybe just drop that line. It may not be an error about the server response.  And just use `throw` not `return Promise.reject`.

Reply:
Removed Promsie.reject and used throw. Also removed the word Response.